runtime/vm/ContinuationHelpers.hpp

```diff
@@ -83,6 +83,36 @@ class VM_ContinuationHelpers {
 		SWAP_MEMBER(oldEntryLocalStorage, J9VMEntryLocalStorage*, threadELS, continuation);
 	}
 
+	/**
+	 * Check if the related J9VMContinuation is mounted to carrier thread
+	 * @param[in] continuation the related J9VMContinuation
+	 * @return true if it is mounted.
+	 */
+	static VMINLINE bool
+	isContinuationMounted(J9VMContinuation *continuation)
+	{
+		return J9_ARE_ANY_BITS_SET(continuation->state, ~(uintptr_t)J9_GC_CONTINUATION_CONCURRENTSCANNING);;
+	}
+
+	static VMINLINE bool
+	isContinuationMountedOrConcurrentlyScanned(J9VMContinuation *continuation)
+	{
+		return continuation->state;
```

Comment:
I don't think perf is critical but it would be nicer to do:

``` (isContinuationMounted(continuation->state) || isConcurrentlyScanned(continuation->state) ```


Perhaps, compiler could even be able to optimize it!?